src/libs/SidebarUtils.js

```diff
@@ -108,7 +108,7 @@ function getOrderedReportIDs(reportIDFromRoute) {
     const isInDefaultMode = !isInGSDMode;
 
     // Filter out all the reports that shouldn't be displayed
-    const reportsToDisplay = _.filter(chatReports, (report) => ReportUtils.shouldReportBeInOptionList(report, reportIDFromRoute, isInGSDMode, currentUserLogin, iouReports, betas, policies));
+    const reportsToDisplay = _.filter({...chatReports, ...iouReports}, (report) => ReportUtils.shouldReportBeInOptionList(report, reportIDFromRoute, isInGSDMode, currentUserLogin, iouReports, betas, policies));
```

Comment:
I think we'll need to update [this conditional](https://github.com/Expensify/App/blob/aa166fd5ae4bb0d194dfc232fc19cc31085fa069/src/libs/SidebarUtils.js#L31) to us [isMoneyRequestReport](https://github.com/Expensify/App/blob/aa166fd5ae4bb0d194dfc232fc19cc31085fa069/src/libs/ReportUtils.js#L147), so we also include Expense reports. Maybe we should also rename `iouReports` to `moneyRequestReports` to reflect this change as well